crates/ruma-federation-api/src/transactions/edu.rs

```diff
@@ -227,13 +228,13 @@ pub struct DeviceListUpdateContent {
 }
 
 impl DeviceListUpdateContent {
-    /// Creates a new `DeviceListUpdateContent` with default `prev_id`, `deleted`
-    /// and `keys` fields.
-    pub fn new(user_id: UserId, device_id: DeviceIdBox, name: String, stream_id: UInt) -> Self {
+    /// Creates a new `DeviceListUpdateContent` with default `prev_id`, `deleted`,
+    /// `device_display_name` and `keys` fields.
```

Comment:
This doc change seems wrong.

Reply:
Note that the comment lists the properties that uses a *default* value (which is the case for `device_display_name` now). What do you think about changing to something like?:

"Create a new `DeviceListUpdateContent` with the given `user_id`, `device_id` and `stream_id`. The rest of the fields will be filled with a default value."

Reply:
Yes, please. Listing the ones that are defaulted is super unusual. If you mention the rest is defaulted (which is not mentioned elsewhere I think) then it should probably be in its own paragraph. (to keep the first one shorter)

Reply:
I've changed this doc and the other doc blocks mentioning default parameters in this file.

I can split the other doc and test changes in a different PR if you prefer it.